### DISCUSSION TITLE
JBD BMS problems with multiple batteries

### DISCUSSION THREAD

**Discussion by Bohne412** [Louisvdw/dbus-serialbattery#1101] [General]
Hello

I've been using dbus battery for a long time with 3 batteries each with 4 cells. So far I have been able to use all 3 batteries on the cerbo at the same time. But now you can only connect one battery. Is there perhaps an idea what I can change to make it work again?

Best regards

**Comment by mr-manuel** [comment]
Check the documentation on how to troubleshoot. There is a specific section for it.

**Comment by Bohne412** [comment]
Bekomme das leider nicht hin . Stehe da leider etwas auf dem Schlauch .
